### PR TITLE
Adds forms and logs outputs

### PR DESCRIPTION
**Pull request by mau-fyi**
Forms redirect the individual to the home page after form submission.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    // Get the value entered in the form.
+    String textValue = request.getParameter("text-input");
+
+    // Print the value so you can see it in the server logs.
+    System.out.println("You submitted: " + textValue);
+
+    // Write the value to the response so the user can see it.
+    response.getWriter().println("You submitted: " + textValue);
+
+    //redirect user back to homepage
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java]: nit: Good to be consistent with the comment formatting. 1 space after the //

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -40,5 +40,24 @@ <h1>Mauricio's Portfolio</h1>
...
       <img id="favorite-pic" style="display: none;">
     </div>
+    <form action="/form-handler" method="POST">
+        <p>Enter some text here:</p>
+        <textarea name="text-input">hello world</textarea>
+        <br/>
+        <input type="submit" />
+    </form>
+    <footer>
+        <form action="/suggestions" method="POST">
```
Comment (sbahaud) [inline portfolio/src/main/webapp/index.html:50]: Where is the suggestions servelet? It is working your website, so I am guessing it is not in this PR?

**Review by sbahaud — COMMENTED** [review]
Thanks for the clear PR description!

Added a minor comment. I will approve once that is resolved

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java (hunk 1 above)
Comment (mau-fyi) [reply portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java]: Got it!

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (mau-fyi) [reply portfolio/src/main/webapp/index.html:50]: Should be added now

Comment (sbahaud) [reply portfolio/src/main/webapp/index.html:50]: This was somehow missed in the commit. It is now in this PR

**Review by sbahaud — APPROVED** [review]
LGTM

**Pull request merged**